Can't build mig on OS X due to undefined variable in netstat module

Trying to upgrade my mig agent on OS X, but it fails during the build:

```
april@anhedonia > ~/.g/s/m/mig > master > make
# test all modules
GOOS=darwin GOARCH=amd64 GO15VENDOREXPERIMENT=1 go test mig.ninja/mig/modules/...
# mig.ninja/mig/modules/netstat
modules/netstat/netstat.go:220: undefined: HasSeenIP
ok  	mig.ninja/mig/modules	0.116s
ok  	mig.ninja/mig/modules/agentdestroy	0.010s
ok  	mig.ninja/mig/modules/example	0.011s
ok  	mig.ninja/mig/modules/file	0.219s
ok  	mig.ninja/mig/modules/memory	0.419s
FAIL	mig.ninja/mig/modules/netstat [build failed]
ok  	mig.ninja/mig/modules/ping	0.010s
ok  	mig.ninja/mig/modules/pkg	0.009s
ok  	mig.ninja/mig/modules/scribe	0.008s
ok  	mig.ninja/mig/modules/timedrift	0.009s
ok  	mig.ninja/mig/modules/upgrade	0.007s
make: *** [test-modules] Error 2
```

I'm on OS X El Capitan, 10.11.3, with go version 1.6.

--- a/modules/netstat/netstat_darwin.go
+++ b/modules/netstat/netstat_darwin.go
@@ -173,3 +173,9 @@ func HasListeningPort(port string) (found bool, elements []element, err error) {
 	err = fmt.Errorf("HasListeningPort() is not implemented on %s", runtime.GOOS)
 	return
 }
+
+func HasSeenIP(val string) (found bool, elements []element, err error) {
+	// XXX Currently not implemented for darwin.
+	err = fmt.Errorf("HasSeenIP(): operation is not implemented on darwin")
+	return
+}

--- a/modules/netstat/netstat_windows.go
+++ b/modules/netstat/netstat_windows.go
@@ -72,3 +72,9 @@ func HasListeningPort(port string) (found bool, elements []element, err error) {
 	err = fmt.Errorf("HasListeningPort() is not implemented on %s", runtime.GOOS)
 	return
 }
+
+func HasSeenIP(val string) (found bool, elements []element, err error) {
+	// XXX Currently not implemented for windows.
+	err = fmt.Errorf("HasSeenIP(): operation is not implemented on windows")
+	return
+}

